index.js

```diff
@@ -1,74 +1,14 @@
-const form = document.querySelector('form');
-const bookList = document.getElementById('bookList');
-
-class Abooks {
-  constructor() {
-    this.books = [];
-
-    this.displayBooks = () => {
-      const storedBooks = JSON.parse(localStorage.getItem('books'));
-      if (storedBooks) {
-        this.books = storedBooks;
-        bookList.className = 'bookList';
-        bookList.innerHTML = this.books
-          .map(
-            (book, index) => `
-        <li class="lineBook">
-          "${book.title.toUpperCase()}" by ${book.author}
-          <button class="btn btn-remove box-shadow" data-index="${index}">Remove</button>
-        </li>
-      `
-          )
-          .join('');
-      }
-    };
-
-    this.submitbtn = () => {
-      const title = document.getElementById('bookTitle').value;
-      const author = document.getElementById('authorName').value;
-      this.books.push({ title, author });
-      localStorage.setItem('books', JSON.stringify(this.books));
-      form.reset();
-      this.displayBooks();
-    };
-
-    this.removeBook = (index) => {
-      this.books.splice(index, 1);
-      localStorage.setItem('books', JSON.stringify(this.books));
-      this.displayBooks();
-    };
-  }
-}
-
-const abooks = new Abooks();
-abooks.displayBooks();
-
-form.addEventListener('submit', (e) => {
-  e.preventDefault();
-  abooks.submitbtn();
-});
-
-bookList.addEventListener('click', ({ target }) => {
-  if (target.classList.contains('btn-remove')) {
-    const { index } = target.dataset;
-    abooks.removeBook(index);
-  }
-});
-
-/* Navigation Bar Links */
-const menuItems = Array.from(document.querySelectorAll('.menu-item'));
-const contentItems = Array.from(document.querySelectorAll('.content'));
-
-menuItems.forEach((item) => {
-  const itemID = item.id;
-  item.addEventListener('click', () => {
-    contentItems.forEach((content) => {
-      const contentId = content.id;
-      if (contentId.includes(itemID)) {
-        content.classList.remove('hide');
-      } else {
-        content.classList.add('hide');
-      }
-    });
-  });
-});
+import './modules/actions.js';
+import { DateTime } from 'luxon';
```

Comment:
- [ ] Please 🙏🏻 always make sure that your code is working without any console error before asking for the review, as I can see that your code has any issue related to the `luxon` library path. It also seems like you used the `parcel` bundler at your local machine, which definitely helped you to run everything without any problem, because any bundler no matter whether `webpack`, `parcel` or `vite`, all bundler manages the dependencies very well during, but as per the requirement you need to use the `luxon` in your project which requires the `relative` path and it seems like no working properly without bundler. The solution for the problem is to use the `luxon` minified version *javascript* file in your modules folder and then you can import the `DateTime` functionality from that javascript and then your code will work properly. Kindly have a look at the below-attached video to see the problem.

## Video
[Screencast from 10-07-23 11:35:41 PM IST.webm](https://github.com/RysthCraft/ES6-Awesome-Books/assets/66476329/60dc7d8a-82db-48b9-987f-7c8c4eb7f497)

### Note

Because of your above-mentioned error, I was not able to add the new books and see the list of the books, but on my local system when I added the `luxon.min.js` file manually, it worked and I was able to add the books, see the list of books and delete the books. Kindly consider the use of the script file of `luxon` and use it. Rest you did a great job.

Reply:
Thanks! @skyv26 

I'm going to fix that now.